build/deploy/mkimage.sh

```diff
@@ -13,19 +13,16 @@ set -ex
 cd -P "$(dirname $0)"
 DIR=$(pwd -P)
 
-rm -rf resources ui cockroach
-mkdir -p build
+rm -rf build && mkdir -p build
 docker run -v "$(pwd)/build":/build "cockroachdb/cockroach-dev" shell "cd /cockroach && \
   rm -rf /build/*
   make clean build testbuild >/dev/null 2>&1 && \
   find . -name '*.test' -type f -printf "\"/build/%h \"" | xargs mkdir -p && \
   find . -name '*.test' -type f -exec mv {} "/build/{}" \; && \
-  cp -r resources /build/resources && \
-  cp -r ui /build/ui && \
+  cp -r {ui,resources} /build/ && \
```

Comment:
Ah, you're copying out of the cockroach-dev container to a local dir and then from the local dir back into a new container. I did this in a project at Square and didn't have any weird permission issues, though I was using boot2docker on Mac OS X.